src/shared/Microsoft.Git.CredentialManager/Authentication/MicrosoftAuthentication.cs

```diff
@@ -205,6 +234,15 @@ private async Task<IPublicClientApplication> CreatePublicClientApplicationAsync(
                 appBuilder.WithParentActivityOrWindow(() => new IntPtr(hWndInt));
             }
 
+            // On Windows 10 & .NET Framework try and use the WAM broker
+            if (enableBroker && PlatformUtils.IsWindows10())
+            {
+#if NETFRAMEWORK
+                appBuilder.WithExperimentalFeatures();
```

Comment:
Experimental? That sounds safe... 
![image](https://user-images.githubusercontent.com/127790/113997468-7573cf00-9858-11eb-95f7-309f2b18c2df.png)

Reply:
@bgavrilMS is this still needed for WAM support to be available?

Reply:
Experimental = public preview :). We're waiting to get some more traction on this feature before moving it to GA. I will discuss directly with Matthew if we make any breaking changes. IMO the only difficulties are around supporting Personal accounts.